etna/lib/helpers.rb

```diff
@@ -3,33 +3,57 @@
 
 module WithEtnaClients
   def environment
-    EtnaApp.instance.environment
+    Etna::EtnaApp.instance.environment
```

Comment:
I think all of these references in `helpers.rb` should just be `EtnaApp` instead of `Etna::EtnaApp` since we've left it out of the Etna module.